eclipsePlugin/src/de/tobject/findbugs/messages_ja.properties

```diff
@@ -1,30 +1,31 @@
-property.addbutton              = \u8ffd\u52a0...
-property.bugCodes               = \u30d1\u30bf\u30fc\u30f3
-property.categoriesGroup        = \u5831\u544a (\u53ef\u8996) \u30d0\u30b0\u30fb\u30ab\u30c6\u30b4\u30ea\u30fc
-property.category               = \u30ab\u30c6\u30b4\u30ea\u30fc
-property.detectorDescription    = \u8aac\u660e
-property.detectorName           = \u30c7\u30a3\u30c6\u30af\u30bf\u30fcID
-property.detectorsTab           = \u30c7\u30a3\u30c6\u30af\u30bf\u30fc\u69cb\u6210
-property.effort                 = \u5206\u6790\u529b
-property.effortdefault          = \u30c7\u30d5\u30a9\u30eb\u30c8
+
+property.addbutton              = \u8FFD\u52A0...

```

Comment:
Also, while I like uppercase A-F in hexadecimal numbers, this makes the change bigger than necessary. If you really want to change lower- to uppercase, please do this in a separate change.